cypress/e2e/transformation.cy.js

```diff
@@ -14,7 +14,7 @@ describe("Report transformation", () => {
 
   it("Update transformation", function () {
     cy.visit("");
-    cy.get("#SettingsButton").click();
+    cy.get("[data-cy-debug-table='openSettings']").click();
```

Comment:
Deze settings button heeft niks te maken met de table
```suggestion
    cy.get("[data-cy-debug-button='settings']").click();
```